src/helpers/analytics.ts

```diff
@@ -21,6 +22,7 @@ import * as search from './analyticsEvents/search';
 import * as openCloseStudyGuides from './analyticsEvents/studyGuides/openClosePopUp';
 import * as openUTG from './analyticsEvents/studyGuides/openUTG';
 import * as unload from './analyticsEvents/unload';
+import { disableAnalyticsCookie } from './constants';
```

Comment:
you can define this in this same file, `helpers` don't really have a common theme so a shared constants file might be confusing

Reply:
But we want to have that `disableAnalyticsCookie` name exported, right?

Reply:
I think linter will yell at you if you export a const that is not used in another file so for now, I think it shouldn't be exported